vignettes/Model.Rmd

```diff
@@ -14,86 +14,282 @@ knitr::opts_chunk$set(
+
+### Parameters
+
+Parameters shown on the infographic include mosquito developmental rates:
+
+-   $r_{EL}$: from early to late larval stage
+
+-   $r_{L}$: from late larval stage to pupal stage
+
+-   $r_P$: from pupal stage to adult stage
+
+Mosquito infection and incubation rates:
+
+-   $\Lambda_M$: the force of infection on mosquitos (FOIM, i.e. from human to mosquito)
+
+-   $r_{EM}$: extrinsic incubation period
+
+And mortality rates:
+
+-   $\mu_E(t)$: the early larval stage death rate
+
+-   $\mu_L(t)$: the late larval stage death rate
+
+-   $\mu_P$: the pupal death rate
+
+-   $\mu_M$: the adult mosquito death rate
+
+Larval mosquitoes experience density dependent mortality due to a carrying capacity, $K(t)$, which may change seasonally with rainfall and where $\gamma$ is the effect of density-dependence on late stage larvae compared with early stage larvae as follows:
+
+$$
+\mu_E = \mu_E^0(1+\frac{E(t)+L(t)}{K(t)}) \\
+\mu_L = \mu_L^0(1+\gamma\frac{E(t)+L(t)}{K(t)})
+$$
+
+## Key Model References (structure and dynamics)
+
+Griffin JT, Hollingsworth TD, Okell LC, Churcher TS, White M, et al. (2010) Reducing *Plasmodium falciparum* Malaria Transmission in Africa: A Model-Based Evaluation of Intervention Strategies. PLOS Medicine 7(8): e1000324. <https://doi.org/10.1371/journal.pmed.1000324>
+
+White, M.T., Griffin, J.T., Churcher, T.S. *et al.* Modelling the impact of vector control interventions on *Anopheles gambiae* population dynamics. *Parasites Vectors* **4**, 153 (2011). <https://doi.org/10.1186/1756-3305-4-153>
+
+Griffin, J., Ferguson, N. & Ghani, A. Estimates of the changing age-burden of *Plasmodium falciparum* malaria disease in sub-Saharan Africa. *Nat Commun* **5**, 3136 (2014). <https://doi.org/10.1038/ncomms4136>
+
+Griffin, J. T., Déirdre Hollingsworth, T., Reyburn, H., Drakeley, C. J., Riley, E. M., & Ghani, A. C. (2015). Gradual acquisition of immunity to severe malaria with increasing exposure. Proceedings of the Royal Society B: Biological Sciences, 282(1801). <https://doi.org/10.1098/rspb.2014.2657>
+
+Griffin, J. T., Bhatt, S., Sinka, M. E., Gething, P. W., Lynch, M., Patouillard, E., Shutes, E., Newman, R. D., Alonso, P., Cibulskis, R. E., & Ghani, A. C. (2016). Potential for reduction of burden and local elimination of malaria by reducing Plasmodium falciparum malaria transmission: A mathematical modelling study. The Lancet Infectious Diseases, 16(4), 465--472. [https://doi.org/10.1016/S1473-3099(15)00423-5](https://doi.org/10.1016/S1473-3099(15)00423-5)
+
+# Basic simulation
+
+### Code
+
+The key package function is `run_simulation()` which, in its most basic form, simply requires a number of timesteps in days. Default parameter settings assume a human population of size of 100, an initial mosquito population size of 1000 with all species in equal proportions, with no treatment interventions and no seasonality. The full parameters list can be seen in the documentation for `get_parameters()`.
```

Comment:
Default parameters have a single species (So species proportion = 1).

Reply:
Is the default mosquito species supposed to be gambiae? They match with gambiae, except for the mum value. Just wondering if this is a mistake should have been updated?

Reply:
I've changed this to say gambiae, and I'm going to create a new pull request to change that value.

Reply:
I think it is a combination of all of them